src/uber/UberGitHubBetaUsers.php

```diff
@@ -0,0 +1,52 @@
+<?php
+
+// UberGitHubBetaUsers
+final class UberGitHubBetaUsers extends Phobject {
+  private $url = self::URL;
+  const URL = 'https://gollo.uberinternal.com/r?request=get-group-members&group=github-beta-users';
+
+  const TIMEOUT_SECONDS = 10;
+
+  public function __construct() {}
+
+  public function getGitHubBetaUsers(): array {
+    try {
+      $usso = new UberUSSO();
+      $hostname = parse_url(self::URL, PHP_URL_HOST);
+      $token = $usso->maybeUseUSSOToken($hostname);
+      if (!$token) {
+        $token = $usso->getUSSOToken($hostname);
+      }
+
+      $future = id(new HTTPSFuture($this->url))
+        ->setFollowLocation(false)
+        ->setTimeout(self::TIMEOUT_SECONDS)
+        ->setMethod('GET')
+        ->addHeader('Authorization', "Bearer {$token}");
+
+      list($body, $headers) = $future->resolvex();
+
+      if (empty($body)) {
+        return [];
+      }
+
+      $resp = phutil_json_decode($body);
+      return $resp['Members'] ?? [];
+    } catch (Exception $e) {
+      return [];
+    }
+  }
+
+
+  public function isCurrentUserInGitHubBetaUsers() {
+    $githubBetaUsers = $this->getGitHubBetaUsers();
+    $user = getenv('USER');
+    if ($user === false) {
```

Comment:
Might be better using `!$user` since it will handle as true if user is `null`

Reply:
```
/**
 * Gets the value of an environment variable
 * @link https://php.net/manual/en/function.getenv.php
 * @param string|null $name <p>
 * The variable name.
 * </p>
 * @param bool $local_only [optional] <p>
 * Set to true to only return local environment variables (set by the operating system or putenv).
 * </p>
 * @return string|array|false the value of the environment variable
 * varname or an associative array with all environment variables if no variable name
 * is provided, or false on an error.
 */
#[Pure(true)]
function getenv(
    #[PhpStormStubsElementAvailable(from: '5.3', to: '7.0')] $varname,
    #[PhpStormStubsElementAvailable(from: '7.1')] ?string $name = null,
    #[PhpStormStubsElementAvailable(from: '5.6')] bool $local_only = false
): array|string|false {}
```

^ according to the function signature for getenv, it will return a string containing the env value or `false` if the env value doesn't exist if we pass in a string. So I think the `$user === false` is what we want here.